README.md

```diff
@@ -51,41 +51,21 @@ git clone git@github.com:twilio-labs/plugin-queued-callbacks-and-voicemail.git
 cd plugin-queued-callbacks-and-voicemail/public && mv appConfig.example.js appConfig.js
```

Comment:
Is there still a need for this command or can we just leave `public/appConfig.example.js` alone now?